src/modules/rep.ts

```diff
@@ -42,9 +42,7 @@ export class RepModule extends Module {
 		const NO_GIVE = '❌';
 
 		// Check for thanks messages
-		const isThanks = this.THANKS_MESSAGES.some(s =>
-			msg.content.toLowerCase().includes(s),
-		);
+		const isThanks = this.THANKS_REGEX.test(msg.content.toLowerCase());
```

Comment:
use /i at the end of the regex instead of `String#toLowerCase`.

Reply:
Fixed